tictactoeSpecific.md

```diff
@@ -0,0 +1,112 @@
+
+
+# TicTacToe Game Specification
+
+## About the Game:
+
+Tic-Tac-Toe, it's a game of two players, who take turns marking spaces (**X or O**), it can be any symbols)in a 3*3 grid.
+
+## Game Rules:
+
+As we said, it's a two-player game, where one player takes one symbol(**X**) and the second player takes one symbol(**O**).
+
+The first player to get 3 of her marks in a row (**up, down, across, diagonally**) is the winner.
+
+When all 9 squares are full, the game is over or draw.
```

Comment:
Over and draw not over or draw. Be specific. If you have doubts get it clarified. Computer cannot decide when it is over and when it is draw if both is possible when all 9 squares are filled. That statement is ambiguous.